### PR TITLE
aruha-185 fix enrichment api documentation

### PR DESCRIPTION
**Pull request by rcillo**
Trying to create an event type without any specified enrichment strategy
result in:

```
Failed to create event type end2end_monitor.test_invoked, code: 422,
message: {u'status': 422, u'type': u'http://httpstatus.es/422',
u'detail': u'must define metadata enrichment strategy', u'title':
u'Unprocessable Entity'}
```

But according to specs default enrichment_strategies is `metadata-enrichment`.

So in this PR we made it clear that enrichment strategy is required, when it
should specify it and why.

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/zalando/nakadi/pull/239" target="_blank">ReviewNinja

File: api/nakadi-event-bus-api.yaml [hunk 1]
```diff
@@ -1217,6 +1221,7 @@ definitions:
       - name
       - category
       - owning_application
+      - enrichment_strategies
```
Comment (v-stepanov) [inline api/nakadi-event-bus-api.yaml]: is it possible to create undefined event-type without enrichment_strategies field

Comment (rcillo) [inline api/nakadi-event-bus-api.yaml]: It is. Actually, this field is required in the case of business and data change events. I don't know how to get rid of these exceptions. This category concept is really hard to deal with.

Comment (v-stepanov) [inline api/nakadi-event-bus-api.yaml]: Then I believe it should not be in a list of required fields.
I think it's ok - we can just say that for certain categories we have additional event-type validation.

Comment (v-stepanov) [inline api/nakadi-event-bus-api.yaml]: But in general I agree. These categories make me think of:

Comment (rcillo) [inline api/nakadi-event-bus-api.yaml]: Then you think I should say in the description that it might be required in special cases?

Comment (v-stepanov) [inline api/nakadi-event-bus-api.yaml]: Yes, I think you can say that it is required for categories business and data.

**Pull request merged**